src/main/java/com/labsynch/labseer/domain/StandardizationDryRunCompound.java

```diff
@@ -618,7 +613,7 @@ public static TypedQuery<StandardizationDryRunCompound> findStandardizationDryRu
 			throw new IllegalArgumentException("The corpName argument is required");
 		EntityManager em = StandardizationDryRunCompound.entityManager();
 		StringBuilder queryBuilder = new StringBuilder(
-				"SELECT o FROM StandardizationDryRunCompound AS o WHERE o.corpName = :corpName");
+				"SELECT o FROM StandardizationDryRunCompound AS o JOIN o.parent p WHERE p.corpName = :corpName");
```

Comment:
nit: Should this use `JOIN FETCH` ?

Reply:
It actually really depends on how you are using it. JOIN FETCH can speed things up but it's not strictly necessary and isn't obvious that its needed in this context.